scripts/e2e/init-template-e2e.js

```diff
@@ -117,9 +117,9 @@ async function initNewProjectFromSource(
 
     console.log('Running react-native init without install');
     execSync(
-      `node ./packages/react-native/cli.js init ${projectName} \
+      `npx @react-native-community/cli@next init ${projectName} \
         --directory ${directory} \
-        --template ${templatePath} \
+        --version next \
```

Comment:
Note: this won't work for `rc.0` the next time.

Let's use it for now to unblock us but this now assumes the template is already published *before* be do the testing which is not the case.

I think the most correct way to do would be to:

1. git clone the template
2. git checkout to the `0.75-stable` branch (i.e. the same branch this script is executed from).
3. Update the template to align `react-native` and all the `@react-native/*` dependencies.
4. Run `npm pack` to create a tarball of the template
5. Run `npx npx @react-native-community/cli@next init ${projectName} --template file://{path-to-packaged-template}`

Reply:
the current approach doesn't work to test older versions of react native.
But also the steps highlighted here don't work, unless we have a release branch for the template as well. 🤔 

The problem is that, when we test RC1, for example, React Native RC1 has not been published yet. And locally, the version of react native has not been updated yet. So:
- we are testing RC1, while the version in React native is still RC0
- we need to automate the creation of template release branches
🤔 🤔 🤔 🤔 🤔

Reply:
We can pass an extra parameter to the script, though... We can:
- pass a new `-v` with the version we want to test
- run set-version to bump everything locally to the right version
- run the Verdaccio to publish the packages in the versions we actually want to test
- Assuming that we are in the release branch and that a release branch with the exact same name exists in the template, we can apply your steps.